.github/workflows/publish-and-test.yml

```diff
@@ -0,0 +1,279 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Publish Docker Images & Run E2E QA on Images
+
+on:
+  push:
+    branches: [develop]
+  release:
+    types: [published]
+  workflow_dispatch:
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: ${{ github.repository }}
+  DOCKER_NAME: ${{ vars.DOCKERHUB_USERNAME }}/openelis-global-2
+  DOCKER_TEST_NAME: ${{ vars.DOCKERHUB_USERNAME }}/openelis-global-2-test
+
+jobs:
+  build-and-push-test-image-backend:
```

Comment:
Just to understand , 
We would want to run test images before we publish the actual Images ??